src/routes/officeRoute.js

```diff
@@ -0,0 +1,5 @@
+import Office from '../controllers/officeController';
+
+module.exports = (app) => {
```

Comment:
use es6 module import/export. Your code need to be consistent.

Reply:
Thanks for the feedback. It has been updated